src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -82,6 +82,11 @@
 
 			<ActionSeparator />
 
+			<ActionButton v-if="canLeaveConversation"
+				:icon="iconArchive"
+				@click.prevent.exact="toggleArchiveConversation">
+				{{ labelArchive }}
+			</ActionButton>
```

Comment:
Since this is a purely moderator option it should go into the moderator menu?

Reply:
I don't think it's only a moderator option.
Everybody should archiv a room to have a cleaner room list. 
But wait, this is not possible with this setup. This needs to be done on a user basis. 
So maybe I don't understand the aim here

Reply:
A per user archiving is way over the top from my pov.